src/project-management/android-app.ts

```diff
@@ -16,9 +16,15 @@
 
 import { FirebaseProjectManagementError } from '../utils/error';
 import * as validator from '../utils/validator';
-import { ProjectManagementRequestHandler, assertServerResponse } from './project-management-api-request';
+import { ProjectManagementRequestHandler, assertServerResponse } from './project-management-api-request-internal';
 import { AndroidAppMetadata, AppPlatform } from './app-metadata';
 
+/**
+ * A reference to a Firebase Android app.
+ *
+ * Do not call this constructor directly. Instead, use
```

Comment:
I think the statement regarding the constructor is redundant since we expose this as an interface. Let's remove that line.